api/proto/teleport/legacy/types/types.proto

```diff
@@ -5984,6 +5984,10 @@ message PluginEntraIDSettings {
 
   // SyncSettings controls the user and access list sync settings for EntraID.
   PluginEntraIDSyncSettings sync_settings = 1;
+
+  // AccessGraphSettings controls settings for syncing access graph specific data.
+  // When this is null, Entra ID integration with Access Graph is disabled.
+  PluginEntraIDAccessGraphSettings access_graph_settings = 2;
```

Comment:
Can't we use the same plugin as `PluginEntraIDSyncSettings`? 
Having so many plugins for the same thing seems abusive

Reply:
This is not a separate plugin, just a further sub-field in the (single) plugin's spec. If we feel badly about nested proto messages for any reason, I can pull it out. Although I kind of like shoving the TAG settings under a different field, especially since they contain the cache with the app info blobs.